doc/schemas/gov_openapi.json

```diff
@@ -1995,29 +1942,6 @@
         }
       }
     },
-    "/gov/members": {
```

Comment:
@lynshi is that something you are relying on at the moment?

Reply:
Yes, we need a way to retrieve the list of members so that we can figure out which identity version to use when connecting to the CCF network - this is required for the edge case where we restart before we perform identity rotation. If after some restart, there's a newer Certificate version in AKV, that version may not be registered in CCF yet, so our operator identity needs to know specifically which version to fetch to use to auth with CCF, and it figures this out by comparing available versions what the public keys in CCF.

Reply:
@lynshi member details can now also be accessed via the newly added endpoints like `/gov/members/certs`, `/gov/members/info`, `/gov/kv/members/encryption_public_keys` [0]. This effort is to remove duplicates and keep the endpoints consistent. For example, `/gov/kv/*` now exposes the governance public tables via endpoints. 

[0] https://github.com/microsoft/CCF/blob/main/doc/schemas/gov_openapi.json#L1558-L1602

Reply:
Good to know! It would still be great to keep these endpoints around (deprecated and undocumented) for now so that code upgrades can go smoothly (since a lot of communication is through a service / load balancer, we might be routed to a node that does not support the endpoint called).

We'd be able to accommodate the eventual removal better this way. If (for example) these endpoints are deprecated in `4.x` and removed in `5.x`, we can code upgrade to `4.x`, update our code to use the new endpoints, and then their removal in `5.x` won't be noticed.

Reply:
Sure, we will mark them as deprecated for now (and undocumented in the Azure API).